lib/beaker-hostgenerator/hypervisor.rb

```diff
@@ -39,12 +39,16 @@ def self.create(node_info, options)
     #                                                and their implementations.
     def self.builtin_hypervisors
       {
-        'vmpooler' => BeakerHostGenerator::Hypervisor::Vmpooler,
-        'vagrant' => BeakerHostGenerator::Hypervisor::Vagrant,
-        'vagrant_libvirt' => BeakerHostGenerator::Hypervisor::Vagrant,
-        'docker' => BeakerHostGenerator::Hypervisor::Docker,
         'abs' => BeakerHostGenerator::Hypervisor::ABS,
+        'container' => BeakerHostGenerator::Hypervisor::Docker,
+        'container_docker' => BeakerHostGenerator::Hypervisor::Docker,
+        'container_swarm' => BeakerHostGenerator::Hypervisor::Docker,
```

Comment:
I am still missing a `container_podman` definition here?

Reply:
added it